examples/sass/_mixins.scss

```diff
@@ -1,9 +1,20 @@
 /* From https://coderwall.com/p/qac-og */
 
 @mixin calc($property, $expression, $x:"") {
-
-
   #{$property}: -moz-calc(#{$expression}) #{$x};
   #{$property}: -webkit-calc(#{$expression}) #{$x};
   #{$property}: calc(#{$expression}) #{$x};
-}
\ No newline at end of file
+}
+
+// Mixin for simple using viewport units with buggyfill
+// Example include: @include viewport-unit((width: 23.5vw, height: 32vw, background-position: -47.8vw top));
+// Its work!)
+
+@mixin viewport-unit($prop-value){
+  $str:;
```

Comment:
Sass developer says "Sass isn’t very strict with variable types"(http://hugogiraudel.com/2013/07/15/understanding-sass-lists/). Edit as you like.